tests/store/test_abstract_store.py

```diff
@@ -0,0 +1,86 @@
+import mock
+
+from mlflow.store.abstract_store import AbstractStore
+from mlflow.entities import ViewType
+
+
+class ConcreteStore(AbstractStore):
+
+    def list_experiments(self, view_type=ViewType.ACTIVE_ONLY):
+        raise NotImplementedError()
+
+    def create_experiment(self, name, artifact_location):
+        raise NotImplementedError()
+
+    def get_experiment(self, experiment_id):
+        raise NotImplementedError()
+
+    def delete_experiment(self, experiment_id):
+        raise NotImplementedError()
+
+    def restore_experiment(self, experiment_id):
+        raise NotImplementedError()
+
+    def rename_experiment(self, experiment_id, new_name):
+        raise NotImplementedError()
+
+    def get_run(self, run_uuid):
+        raise NotImplementedError()
+
+    def update_run_info(self, run_uuid, run_status, end_time):
+        raise NotImplementedError()
+
+    def create_run(self, experiment_id, user_id, run_name, source_type, source_name,
+                   entry_point_name, start_time, source_version, tags, parent_run_id):
+        raise NotImplementedError()
+
+    def delete_run(self, run_id):
+        raise NotImplementedError()
+
+    def restore_run(self, run_id):
+        raise NotImplementedError()
+
+    def get_metric_history(self, run_uuid, metric_key):
+        raise NotImplementedError()
+
+    def search_runs(self, experiment_ids, search_filter, run_view_type):
+        raise NotImplementedError()
+
+    def log_batch(self, run_id, metrics, params, tags):
+        raise NotImplementedError()
+
+
```

Comment:
Would be good to write tests for non-abstract methods in `AbstractStore`. You have covered `log_*` methods--can we write one for `list_run_infos` as well?

Reply:
Done - also added tests for `get_experiment_by_name` to complete the set.